addon/components/o-s-s/array-input.ts

```diff
@@ -59,7 +66,8 @@ export default class OSSArrayInput extends Component<OSSArrayInputArgs> {
 
   @action
   keyListener(keyboardEvent: KeyboardEvent): void {
-    if (keyboardEvent.code === 'Enter' && this.currentValue.length > 0) {
+    if (this.keyboardTriggers.includes(keyboardEvent.code) && this.currentValue.length > 0) {
```

Comment:
are you sure you want to check against codes rather than keys ?

Reply:
the arg is like `['Space']`, I thought it was better like that and more "human"

Reply:
After discussing live, we chose to use the `code` instead of `key`
For reference, from the MDN :

> For example, the code returned is “KeyQ” for the Q key on a QWERTY layout keyboard, but the same code value also represents the ' key on Dvorak keyboards and the A key on AZERTY keyboards. That makes it impossible to use the value of code to determine what the name of the key is to users if they’re not using an anticipated keyboard layout.
To determine what character corresponds with the key event, use the KeyboardEvent.key property instead.